charts/k8s-watcher/templates/network-mapper/cluster-role.yaml

```
@@ -0,0 +1,35 @@
+{{- if and .Values.network_mapper ((.Values.network_mapper).enabled | default false) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "otterize.mapper.fullName" . }}
+rules:
+  - apiGroups:
+      - ''
+    resources:
+      - 'endpoints'
+      - 'pods'
+    verbs:
+      - 'get'
+      - 'list'
+      - 'watch'
+  - apiGroups:
+      - 'apps'
+    resources:
+      - 'deployments'
+      - 'daemonsets'
+      - 'replicasets'
+      - 'statefulsets'
+    verbs:
+      - 'get'
+{{ if or (and (eq .Values.network_mapper.global.allowGetAllResources nil) .Values.network_mapper.allowGetAllResources) .Values.network_mapper.global.allowGetAllResources}}
```

Can be simplified: (do check the syntax, i freehanded it)
```suggestion
{{ if or (((.Values.network_mapper).global).allowGetAllResources) (((.Values.network_mapper).global).allowGetAllResources}}
```

---

On second thought - you are right (+1)